R/server.R

```diff
@@ -0,0 +1,305 @@
+server = function(input, output, session) {
+  #####################################
+  # Info diplayed below Sdg image
+  #####################################
+  observe({
+    req(input$country)
+    indicators_country <- load_indicators_map() %>% filter(country_code == input$country)
+    output$info_indicators <- renderUI({ 
+    
+     
+      HTML(paste0(
+        "<br><br><b>Indicators: </b>",indicators_country$indicator_num, "<br>",
+        "<b>Datasets: </b>",indicators_country$indicator_num, "<br>",
+        "<b>Population Definitions: </b>",indicators_country$indicator_num, "<br>",
+        "<b>Description: </b>", indicators_country$countries_name, "<br><br>",
+        "<b>Source: </b>",indicators_country$indicator_num, "<br>"
+        
+      ))
+    
+    })
+  })
+  #####################################
+  # navbar Country 
+  #####################################
+  
+  observe({
+    req(input$country)
+    indicators_country <- load_indicators_map() %>% filter(country_code == input$country)
+    output$navbar_country <- renderUI({ 
+      HTML(paste0(
+        "<b>", toupper(indicators_country$countries_name), "</b>"
+      ))
+    })
+  })
+
+  #####################################
+  # Images SDG
+  #####################################
+  observe({
+    req(input$country)
+    indicators <- load_indicators(input$country)
```

Comment:
If you load all indicators once at the start, you could save a database call here by using the existing indicators dataframe and then using subset() to get the ones for the chosen country.